src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -5779,28 +5759,29 @@ namespace SimAirServingZones {
 						// and so the zone-level Voz must be prorated so their sum equals the system level Vot
 						Ratio = 1.0;
 						if ( VozSumClgBySys( AirLoopNum ) > 0 ) Ratio = VotClgBySys( AirLoopNum ) / VozSumClgBySys( AirLoopNum );
-						for ( ZonesCooledNum = 1; ZonesCooledNum <= NumZonesCooled; ++ZonesCooledNum ) {
-							CtrlZoneNum = AirToZoneNodeInfo( AirLoopNum ).CoolCtrlZoneNums( ZonesCooledNum );
-							FinalZoneSizing( CtrlZoneNum ).MinOA = Ratio * FinalZoneSizing( CtrlZoneNum ).VozClgByZone;
+						for ( int ZonesCooledNum = 1; ZonesCooledNum <= NumZonesCooled; ++ZonesCooledNum ) {
+							int TermUnitSizingIndex = AirToZoneNodeInfo( AirLoopNum ).TermUnitCoolSizingIndex( ZonesCooledNum );
+							TermUnitFinalZoneSizing( TermUnitSizingIndex ).MinOA = Ratio * TermUnitFinalZoneSizing( TermUnitSizingIndex ).VozClgByZone;
 						}
```

Comment:
@EnergyArchmage @rraustad Question:
The above lines adjust `.MinOA` by the ratio of VotClgBySys/VozSumBySys. This adjustment is made here in `UpdateSysSizing` but not redone in `ManageSystemVentilationAdjustments`. 

`FinalZoneSizing( CtrlZoneNum ).MinOA` is reported in the eio zone sizing data as the calculated outdoor air flow, and it's reported in the HVAC Sizing Summary Table as the Minimum Outdoor Air Flow Rate by zone.
Looking at current develop, 5ZoneAirCooled_VRPSizing, HVAC Sizing Summary, Space 1-1 is currently showing 0.142 for Minimum Outdoor Air Flow Rate. I think this is wrong, because I don't see 0.142 showing up anywhere in the Std 62 report, and the sum across all zones is only 0.621 while the system OA for cooling is 1.0498 for cooling and 0.3149 for heating.
It's 0.142 in v8.8.0 also.

My changes on this branch caused diffs in the HVAC Sizing Summary Minimum Outdoor Air Flow Rate because these changes leave `FinalZoneSizing( CtrlZoneNum ).MinOA` as it was (designspecOA amount divided by smaller of heating and cooling air distribution efficiency.). So, with this branch, it's reporting the same values as Voz-clg (which are the same as Voz-htg).

So, would you agree that we need to recalculate this in `ManageSystemVentilationAdjustments`?
And if there are more than one system serving a zone, then this should report the total OA from all systems to that zone? Oh, but what about zone equipment? That won't get captured here.

Reply:
I would agree that if the answer is wrong now then we need to correct it. @EnergyArchmage does this ring any bells?

Reply:
The intent in ManageSystemVentilationAdjustments is to only adjust *system* level VRP outcomes.  The zone level ventilation outcomes are already set and done and could not be redone.  Doing so would get circular and you'd have to iterate, so no I don't think it is possible to also recalculate.  The only zone level changes I knowingly made was to take out a Population term in the reported zone Voz values (in Std 62.1 tables) because that is how the standard defines Voz.  I took pains to not change ZoneSizing.MinOA.  

I never did fully understand the ratio and scaling adjustments done in the code you point to.  I think it is a mistake but changing zone-level outcomes was out of scope for that project and I didn't go that far.  For example the code comment below is just not the case.  
"//The system-level Vot will always be larger than the sum of the zone Voz
// and so the zone-level Voz must be prorated so their sum equals the system level Vot"
VRP has the advantage of Vot often comes out to be less than the zone sum (idea is that recirculation of somewhat unused fresh air (from non-critical zones) has a benefit that reduces the system level Vot compared to the simple zone sum. )  Sizing the terminal unit to max of heating and cooling makes a lot of sense but the ratio scaling done there doesn't make sense to me.  When Vot is lower, that will scale the TermUnitFinalZoneSizing.MinOA down, not up.  Also the system level Vot is not very accurate at this point in the calling order so doing powerful things with it is scary.